releases/net471/KnownIssues/523633 - Setup - OS upgrade to Windows 10 gets the product in bad state.md

```diff
@@ -0,0 +1,23 @@
+# .NET Framework 4.7.1 installation gets into a bad state on upgrading from Windows 10 Anniversary Update
```

Comment:
I agree with Rich that this title is not especially descriptive. Better would be "Application crashes occur when running on the .NET Framework 4.7.1 after upgrading from Windows 10 Anniversary Update to Windows 10 Creators Update"

Reply:
Will fix